tests/file_storage/tests.py

```diff
@@ -801,6 +804,10 @@ def test_stringio(self):
         with temp_storage.open('tests/stringio') as f:
             self.assertEqual(f.read(), b'content')
 
+    def test_callable_storage_param(self):
+        obj = Storage.objects.create()
```

Comment:
The happy path is ok. What if you pass anything but a Storage object?

Reply:
Ive added a "test_callable_base_class_error_raises" test for when the callable is not a sub of Storage